.golangci.yml

```diff
@@ -114,7 +114,7 @@ linters:
     - revive
     - paralleltest
     - gocognit
-    - errcheck
+    #- errcheck
```

Comment:
TODO:
```suggestion
```

Reply:
@artem I made the same changes as suggested

Reply:
Let's just remove this linter from this list instead of commenting it.